src/app/admin/edit-product-form/index.tsx

```diff
@@ -0,0 +1,129 @@
+'use client'
+
+import {
+  Box,
+  Checkbox,
+  Container,
+  createStyles,
+  MultiSelect,
+  NumberInput,
+  Paper,
+  Select,
+  Stack,
+  Text,
+  TextInput,
+  Textarea,
+} from '@mantine/core'
+import { isInRange, isNotEmpty, useForm } from '@mantine/form'
+import React from 'react'
+import { SubmitButton } from '@/components'
+
+export default function EditProductForm() {
+  const form = useForm({
+    initialValues: {
+      itemName: '',
+      displayName: '',
+      priceValue: 0.0,
+      itemSelect: '',
+      familySelect: '',
+    },
+    validate: {
+      itemName: isNotEmpty('Please enter an item name.'),
+      displayName: isNotEmpty('Please enter a display name.'),
+      priceValue: isInRange({ min: 0.01, max: 1000.0 }, 'Please enter a valid price.'),
+      itemSelect: isNotEmpty('Please select a type.'),
+      familySelect: isNotEmpty('Please select a family category.'),
+    },
+  })
+  return (
+    <>
+      <Container>
+        <Paper radius="md" bg="#FFF5F5" p="xl">
+          <Box>
+            <Text align="center" fw={700}>
+              Edit/Create Product Form
```

Comment:
Can remove "Form". The verbiage is mostly for us devs. The Title doesn't need it.